### PR TITLE
Install xarray from main 

### PR DESCRIPTION
**Pull request by jsignell**
Noticed while setting up my env. Now that [#8872](https://github.com/pydata/xarray/pull/8872) has merged

### CODE REVIEW

**Comment by TomNicholas** [comment]
Thanks! FYI we should be able to point to an actual xarray release very soon (hopefully later today) https://github.com/pydata/xarray/issues/9018

**Comment by TomNicholas** [comment]
We should also remove the note in the `usage.md` page in the docs about using a specific branch of xarray

https://virtualizarr.readthedocs.io/en/latest/usage.html#manual-concatenation-ordering

File: docs/usage.md [hunk 1]
```diff
@@ -246,7 +242,7 @@ combined_vds['air'].data.manifest.dict()
 The keyword arguments `coords='minimal', compat='override'` are currently necessary because the default behaviour of xarray will attempt to load coordinates in order to check their compatibility with one another. In future this [default will be changed](https://github.com/pydata/xarray/issues/8778), such that passing these two arguments explicitly will become unnecessary.
 ```
 
-The general multi-dimensional version of this contatenation-by-order-supplied can be achieved using `xarray.combine_nested`.
+The general multi-dimensional version of this concatenation-by-order-supplied can be achieved using `xarray.combine_nested`.
```
Comment (jsignell) [inline docs/usage.md:245]: Seems like concatenation is an annoying word to spell 🤣

**Comment by TomNicholas** [comment]
Thanks @jsignell !

**Pull request merged**